pallets/pool-system/src/benchmarking.rs

```diff
@@ -50,8 +50,9 @@ benchmarks! {
 		T::AccountId: EncodeLike<<T as frame_system::Config>::AccountId>,
 		<<T as frame_system::Config>::Lookup as sp_runtime::traits::StaticLookup>::Source:
 			From<<T as frame_system::Config>::AccountId>,
-		T::NAV: PoolNAV<T::PoolId, T::Balance, RuntimeOrigin = T::RuntimeOrigin, ClassId = u64>,
+		T::NAV: PoolNAV<T::PoolId, T::Balance, RuntimeOrigin = T::RuntimeOrigin>,
 		T::Permission: Permissions<T::AccountId, Ok = ()>,
+		<T::NAV as PoolNAV<T::PoolId, T::Balance>>::ClassId: From<u16>,
```

Comment:
Why is this needed if we don't use class id anymore? Or do we?

Reply:
I've tried to not touch `PoolNAV` trait, in order to be updated in #908. By now `ClassId` represents the `ItemId` in the `pallet-loans-ref` implementation. We need this extra bound to be able to create them in the benchmark.